Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/BaseListboxController.java

```diff
@@ -378,6 +379,27 @@ public void onEvent(Event event) throws Exception {
         launchCalendar(selectedItem.getName(), selectedItem.getId());
       }
     });
+    
+    this.listBox.getListhead().addEventListener("onColSize", new EventListener<ColSizeEvent>() {
+		@Override
+		public void onEvent(ColSizeEvent event) throws Exception {
+			try {
+				int widthSetByClient = Integer.parseInt((event.getWidth().substring(0,event.getWidth().indexOf("px"))));
+				int newWidth = 0;
+				Listheader listHeader = (Listheader) event.getColumn();
+				if (event.getColIndex() == 0) {
+					newWidth = widthSetByClient <= 40 ? 40 : widthSetByClient;
```

Comment:
Not a change request, just a suggestion: `newWidth = Math.max(widthSetByClient, 40)`.